### PR TITLE
Update relevant files in docker-edgex-consul for Go 1.13.

### LINKED ISSUE #22: Update Minimum Go Version to 1.13
Update references for previous versions of Go to 1.13 in the following files

- Dockerfile
- go.mod
- Jenkinsfile

### PR DESCRIPTION
**Pull request by akramtexas**
Fix #22 
- The file `go.mod` [was already using](https://github.com/edgexfoundry/docker-edgex-consul/blob/master/go.mod) `go 1.13` so it was not modified.
- And the file `Jenkinsfile ` was was not modified after confirming with `DevOps`. The comment below is for a separate project (`docker-edgex-consul`) but with the _same_ situation as here, and applies here equally:
```
- Ernesto Ojeda  3:15 PM
edgeXBuildDocker depends only on the Dockerfile in the repo. It only cares about doing a docker build . So for docker-edgex-mongo, the only thing you would have to update is this file: https://github.com/edgexfoundry/docker-edgex-mongo/blob/master/cmd/Dockerfile#L19 (edited) 
```

### CODE REVIEW

**Review by lenny-goodell — APPROVED** [review]
LGTM

**Comment by lenny-goodell** [comment]
@michaelestrin , I am not a committer on this repo. You are, so you will need to do the merge. Note that the failing verify is the old pipeline that is not longer blocking merge.

**Review by michaelestrin — APPROVED** [review]
LGTM based on Lenny's review.

**Pull request merged**
